### ISSUE TITLE
Name "Pod::Usage::Formatter" used only once

### ISSUE THREAD

**Issue by alanmels** [webmin/webmin#1514]
Didn't touch for years a Webmin server running on CentOS 6.x so long without a single problem. However, since Webmin's UI didn't accept the entered password giving:

```
 Warning!
Login failed. Please try again.
```

I tried to change the root password with:

```
/usr/libexec/webmin/changepass.pl /etc/webmin root newpassword
```

and unfortunately it's giving:

```
Name "Pod::Usage::Formatter" used only once: possible typo at /usr/libexec/webmin/bin/webmin line 6.
No comma allowed after filehandle at /usr/libexec/webmin/bin/passwd line 76.
Error: 
```

Changing the root password with `passwd` and rebooting didn't have any effect on UI's behavior that keeps refusing the correct root password. The `/etc/webmin/webmin.acl` file is showing the `root` user has all the needed access permissions. Any way to temporary gain access over Webmin's UI?

**Comment by alanmels** [comment]
For other Webmin users on CentOS 6.x seeing the same problem. Until Webmin team fixes the above reported error, you can replace the content of the `/etc/webmin/miniserv.users` with:

```
root:x:0
```

restart webmin and it will finally accept your newly set root password.

**Comment by iliarostovtsev** [comment]
Hi,

This issue has been fixed with Webmin 1.980.
